src/content/unsupported-browser/components/supported-web-browser/SupportedWebBrowser.scss

```diff
@@ -0,0 +1,18 @@
+@import 'src/styles/common';
+
+.container {
+  display: flex;
```

Comment:
these wont be supported in older browser such as IE9 (see screenshot in one of the comment). Probably best to use the old float styling or display: block and aligning the images.